identity/client/e2e/tests/login.spec.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import { expect } from "@playwright/test";
+import { loginTest as test } from "../text-fixtures";
+import { Paths } from "../../src/components/global/routePaths";
+import { relativizePath } from "../../src/utility/relativizePaths";
```

Comment:
I don't think it's a good idea to share application code between E2E tests. You store the pathes in the page objects I believe

Reply:
In this case I created both of these to the use on the E2E tests, just put them on other places because I thought they could be reusable, but I agree on the sharing code, and will move them.

Regarding the paths, I'm not sure I understand what you mean, but for the routes available for Identity the way they're set up is not easy to extract just the string needed, so I'd rather have something dedicated for that to avoid unnecessary complexity.